python/emokit/packet.py

```diff
@@ -81,9 +81,8 @@ def __repr__(self):
         """
         Returns custom string representation of the Emotiv Packet.
         """
-        return 'EmotivPacket(counter=%i, battery=%i, gyro_x=%i, gyro_y=%i, gyro_z=%s)' % (
```

Comment:
While I really like that you made a test, I'm thinking maybe we could solve this doing a 'EmotivPacket(counter={}, battery={}, gyro_x={}, gyro_y={}, gyro_z={})'.format(

Instead, which should just display None.